app/controllers/businessmatching/updateservice/ChangeBusinessTypesController.scala

```diff
@@ -52,7 +52,7 @@ class ChangeBusinessTypesController @Inject()(
           .getOrElse(InternalServerError("Unable to show the page"))
   }
 
-  def post() = Authorised.async {
+  def post(edit: Boolean = false) = Authorised.async {
```

Comment:
No edit needed here as this page can't be 'edited'.

Reply:
Edit functionality removed